classes/warrior.lua

```diff
@@ -1,17 +1,31 @@
 local AddonName, SAO = ...
 
 local function registerClass(self)
+    local tasteforblood = GetSpellInfo(60503);
+    local overpower = GetSpellInfo(7384);
+    local execute = GetSpellInfo(5308);
+    local revenge = GetSpellInfo(6572);
+    local victoryrush = GetSpellInfo(34428);

```

Comment:
Please rename to `victoryRush` (in camelCase).

Reply:
Fixed